condainstall.sh

```diff
@@ -26,7 +26,7 @@ fi
 
 # Default conda deps
 if [[ -z $CONDA_DEPS ]]; then
-    CONDA_DEPS='scipy matplotlib pyyaml numpy astropy gammapy healpy astropy-healpix'
+    CONDA_DEPS='scipy matplotlib pyyaml numpy astropy gammapy==0.17 healpy astropy-healpix'
```

Comment:
I would change this to ``gammapy>=0.18``

Reply:
Actually, I want to remove this file in favor of the environment.yml file.

Reply:
Ah okay! That's even better :-)